### PR TITLE
2240 - Fix lower toolbar was working inverse with application menu

### LINKED ISSUE #2240: Application Menu Personalized Role Component: Keyboard Is reversed on lower toolbar
**Describe the bug**
Application Menu Personalized Role Component: Keyboard Is reversed on lower toolbar

*edit*: See the comment below on how to address this 💬 

**To Reproduce**
1. Go to URL: http://4190-beta0-enterprise.demo.design.infor.com/components/applicationmenu/example-personalized-role-switcher.html
2. click on "Find a Coworker"
3.  tab down to the settings
4. arrow left and right
5. Nothing happens with you select the left and and right arrow key. The user should be able to right arrow key to the "Proxy a user" field box

**Expected behavior**
The left and right arrow key should allow the user to highlight the "Proxy a user" and "About this App" 

**Version**

- ids-enterprise: [e.g. v4.9.0 or v4.10.0]

**Screenshots**
Application Menu Error

**Platform**
 - OS Version: [ iOS 8]
 - Browser Name [Chrome]
**Additional context**

### PR DESCRIPTION
**Pull request by deep7102**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Fixed lower toolbar was working inverse with Application Menu.

**Related github/jira issue (required)**:
Closes #2240

**Steps necessary to review your pull request (required)**:
- Pull this branch and build/run the demo app
- Navigate to http://localhost:4000/components/applicationmenu/example-personalized-role-switcher.html
- Click on `Find a Coworker`
- Tab down to the settings
- Press arrow left or right
- Should work fine

**Included in this Pull Request**:
- [x] A note to the change log.

### CODE REVIEW

**Comment by tmcconechy** [comment]
Suggestion: Is there no visual tests for these two pages? I was expecting those to fail. Maybe add two visual regression tests ?

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
 ### v4.20.0 Fixes
 
 - `[Alerts]` Removed dirty tracker from the page due to layout issues. ([#1679](https://github.com/infor-design/enterprise/issues/1679))
+- `[App Menu]` Fixed an issue where lower toolbar was working inverse. ([#2240](https://github.com/infor-design/enterprise/issues/2240))
```
Comment (davidcarlsonberg) [inline docs/CHANGELOG.md]:
```suggestion
- `[App Menu]` Fixed an issue where the lower toolbar inverts left and right keyboard actions. ([#2240](https://github.com/infor-design/enterprise/issues/2240))
```

**Comment by deep7102** [comment]
Added now, visual regression tests.

**Comment by tmcconechy** [comment]
@EdwardCoyle You good with this now?

**Comment by tmcconechy** [comment]
@EdwardCoyle This has visual regression tests now so i think good to go?

**Pull request merged**